etl/steps/data/meadow/ggdc/2022-11-28/penn_world_table.meta.yml

```diff
@@ -1,18 +1,31 @@
 dataset:
```

Comment:
I don't understand: Do you have the same metadata in meadow and garden? In that case you could delete the metadata yaml file in garden, and simply copy the meadow metadata.

Reply:
It will be different when I add the transformations in this step.